test/e2e/framework.go

```diff
@@ -1245,24 +1245,31 @@ func (data *TestData) mutateAntreaConfigMap(mutatingFunc func(data map[string]st
 func (data *TestData) gracefulExitAntreaController(covDir string) error {
 	antreaController, err := data.getAntreaController()
 	if err != nil {
-		return fmt.Errorf("error when getting antrea-controller Pod: %v", err)
+		return fmt.Errorf("error when getting antrea-controller Pod, err: %v", err)
```

Comment:
I appreciate you trying to uniformize the error reporting, but when we just wrap an error in another error, we always use `fmt.Errorf("some message: %v", err)`. So I'm not sure what's motivating this change.

Reply:
I saw  https://github.com/vmware-tanzu/antrea/blob/master/test/e2e/framework.go#L1112 has 
_return fmt.Errorf("nc stdout: <%v>, stderr: <%v>, err: <%v>", stdout, stderr, err)_

Now changed the error reporting as below if there is no 'stderr' in the message:
_fmt.Errorf("error when getting antrea-controller Pod: %v", err)_

the below if there is also 'stderr' in the message:
_fmt.Errorf("cannot retrieve content of file '%s' from Node '%s', stderr: <%v>, err: <%v>", fileName, masterNodeName(), stderr, err)_

Is this the normal way for the error reporting?

Reply:
Yes it seems to be the convention we have adopted in the e2e test files. `fmt.Errorf("error when getting antrea-controller Pod: %v", err)` is the standard way of wrapping an error in Go.